gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/PegasusPlugin.java

```diff
@@ -913,6 +921,41 @@ protected void configureGeneratedSourcesAndJavadoc(Project project)
     }
   }
 
+  protected void configureExtensionSchemaValidation(Project project, SourceSet sourceSet)
+  {
+    // extension schema  directory
+    File extensionSchemaDir = project.file(getExtensionSchemaPath(project, sourceSet));
+
+    if (SharedFileUtils.getSuffixedFiles(project, extensionSchemaDir, PDL_FILE_SUFFIX).isEmpty())
+    {
+      return;
```

Comment:
add some debug?

Reply:
This task should be run during build time by default. If there is no extension schema, it will not run. Do you think adding debug info here will confuse people if they don't use ER.